hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/schema/GetDdlTest.java

```diff
@@ -44,16 +45,16 @@ public void when_queryMappingFromRelationNamespace_then_success() {
         createMapping("a", int.class, int.class);
 
         assertRowsAnyOrder("SELECT GET_DDL('relation', 'a')", ImmutableList.of(
-                new Row("CREATE OR REPLACE EXTERNAL MAPPING \"hazelcast\".\"public\".\"a\" EXTERNAL NAME \"a\" (\n" +
-                        "  \"__key\" INTEGER EXTERNAL NAME \"__key\",\n" +
-                        "  \"this\" INTEGER EXTERNAL NAME \"this\"\n" +
-                        ")\n" +
-                        "TYPE \"IMap\"\n" +
-                        "OPTIONS (\n" +
-                        "  'keyFormat'='java',\n" +
-                        "  'keyJavaClass'='int',\n" +
-                        "  'valueFormat'='java',\n" +
-                        "  'valueJavaClass'='int'\n" +
+                new Row("CREATE OR REPLACE EXTERNAL MAPPING \"hazelcast\".\"public\".\"a\" EXTERNAL NAME \"a\" (" + LE +
```

Comment:
AssertJ has `isEqualToNormalizingNewlines`. It would simplify such tests. In addition we could have `scalar(SqlResult)` helper method. Can be static or even as a default method in `SqlResult`. I created it once for convenience: https://github.com/hazelcast/hazelcast/pull/23140/files#diff-ae948b2b986ea1c9536e8266a88d84a2c31535f5eca06ff372520046b62b4df4R101

Reply:
I'd keep it as a separate issue and introduced later. I guess, just fixing test failure is enough.